docs/manual/workflow-steps/azure.md

```diff
@@ -0,0 +1,202 @@
+# Azure Job Steps
+
+Azure is Microsoft's web services product. It is a cloud computing platform that offers a variety of services including but not limited to storage, networking and analytics. Azure's integration with Rundeck allows you to edit and utilized your Azure services in a Rundeck job. 
+
+## Azure / Storage / Copy
+
+This job step allows users to copy files to and from their Azure storage. 
+
+![Azure - Copy](~@assets/img/azure_copy.png)
+
+- **Storage Account**
+: the name of the Azure storage account you wish to use
```

Comment:
What does this mean?  How can we help them understand better what to fill in here?